### PR TITLE
Remove vue-cli

### PR DESCRIPTION
**Pull request by ChaituVR**

### CODE REVIEW

**Comment by bonustrack** [comment]
Also would be great to have optional chaining `like?.this?`, but i can't make it work with your PR.

**Comment by bonustrack** [comment]
I use 
```js
const a = {
  b: {}
}

if (a.b?.c?) {
  console.log('Hi!');
}
```

**Comment by ChaituVR** [comment]
> Deployment fail with this
> image

@bonustrack  this is fixed now.. i can able to deploy to Heroku without any issues now, can u try deploying it again?

**Comment by bonustrack** [comment]
Deployment works now, great!

File: package.json [hunk 1]
```diff
@@ -46,19 +42,20 @@
...
-    "@vue/cli-plugin-router": "^4.4.0",
-    "@vue/cli-plugin-vuex": "^4.4.0",
     "@vue/eslint-config-prettier": "^6.0.0",
     "@vue/eslint-config-typescript": "^5.0.2",
     "eslint-plugin-prettier": "^3.1.3",
     "eslint-plugin-vue": "^6.2.2",
-    "prettier": "^1.19.1",
-    "vue-template-compiler": "^2.6.11"
+    "nodemon": "^2.0.7",
+    "prettier": "^1.19.1"
```
Comment (bonustrack) [inline package.json:56]: @ChaituVR Can you also remove @vue/eslint-config-prettier and @vue/eslint-config-typescript?

Comment (ChaituVR) [reply package.json:53]: @bonustrack Removed them and all other vue related lint packages, seems to be working the same as before, please let me know if something is missing :)

**Comment by bonustrack** [comment]
Optional chaining doesn't seem to work for me
image

**Review by bonustrack — APPROVED** [review]
All good!

**Pull request merged**